hack/svc/multi-stream.py

```diff
@@ -1,35 +1,38 @@
+import argparse
 import re
 import shutil
-import sys
 import time
 from subprocess import Popen
 
+parser = argparse.ArgumentParser()
+parser.add_argument('-k', '--keys', nargs='+', type=str, dest='alist',
+                    help='<Required> Sets streaming keys', required=True)
 
-def ffmpeg_cmd(streamLink: str, streamKey: str) -> str:
+
+def ffmpeg_cmd(stream_link: str, stream_key: str) -> str:
     process = ["ffmpeg"]
     isUrl = re.match(
-        "^(http[s]?:\/\/(www\.)?|ftp:\/\/(www\.)?|www\.){1}([0-9A-Za-z-\.@:%_\+~#=]+)+((\.[a-zA-Z]{2,3})+)(\/(.)*)?(\?(.)*)?", streamLink)
+        "^(http[s]?:\/\/(www\.)?|ftp:\/\/(www\.)?|www\.){1}([0-9A-Za-z-\.@:%_\+~#=]+)+((\.[a-zA-Z]{2,3})+)(\/(.)*)?(\?(.)*)?", stream_link)
     if isUrl:
         process += ["-reconnect", "1", "-reconnect_at_eof",
                     "1", "-reconnect_delay_max", "10"]
 
-    process += ["-re", "-y", "-i", streamLink,
+    process += ["-re", "-y", "-i", stream_link,
                 "-codec:a", "aac", "-c:v", "copy", "-f", "flv", "-flvflags", "no_duration_filesize",
-                f"rtmp://0.0.0.0:1935/live/{streamKey}"]
+                f"rtmp://0.0.0.0:1935/live/{stream_key}"]
 
     return " ".join(process)
 
 
 def main() -> int:
     if shutil.which("ffmpeg") is None and shutil.which("ffprobe") is None:
         raise Exception("ffmpeg and ffprobe is required")
-    strims = {"streams": ["http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerBlazes.mp4",
-                          "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerBlazes.mp4"],  "keys": sys.argv[1:]}
+    test_url = "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerBlazes.mp4"
     strims_to_open = []
 
-    for s in range(0, len(strims["streams"])):
-        strims_to_open.append(ffmpeg_cmd(
-            strims["streams"][s], strims["keys"][s]))
+    for stream_key in parser.parse_args().alist:

```

Comment:
before this loop lets parse the args like `args = parser.parse_args` and create a flag for the test-url (defaulting to the one you have, and accepting multiple args like you do for stream keys)

Reply:
oh maybe I'm missing the point of this. So if we provide multiple stream keys, that many streams will be creating using the test url?